### DISCUSSION TITLE
Host to Host ipsec certificate based connection between strongswan and windows

### DISCUSSION THREAD

**Discussion by dewilllll** [strongswan/strongswan#1092] [Q&A]
Hi,

I am trying to establish host-to-host IPSec certificate-based authentication between windows and strongswan.
And I am getting "ignoring IKE_SA setup from 192.168.100.70, peer too aggressive" this error 

[community.txt](https://github.com/strongswan/strongswan/files/8890450/community.txt)

![AUTH method](https://user-images.githubusercontent.com/107111530/173340239-157fec31-0602-45df-816c-6348989e7632.png)
![AUTH method2](https://user-images.githubusercontent.com/107111530/173340246-23a0b7d0-5486-4488-a720-660956767a1f.png)
[community.txt](https://github.com/strongswan/strongswan/files/8890443/community.txt)
![ESP proposal ](https://user-images.githubusercontent.com/107111530/173340260-48177ad0-a98f-4a6e-a152-cf1af4fd610e.png)
![Ike key exchange proposal](https://user-images.githubusercontent.com/107111530/173340262-f526c850-8e49-43da-a349-c1281d96e27f.png)

**Comment by tobiasbrunner** [comment]
As you can see in the log, the client just sends new IKE_SA_INIT requests. Why it would do that (and not at least retransmits of previous messages) or why it would not process any of the IKE_SA_INIT responses, no idea. I guess you have to ask somebody who knows Microsoft products.
